Source/Fuse.GeoLocation/iOS/IOSLocationProvider.uno

```diff
@@ -29,7 +33,12 @@ namespace Fuse.GeoLocation
 			error: ^void (NSError* err)
 			{
 				@{IOSLocationProvider:Of(_this).OnError(string):Call(err.localizedDescription)};
-			}];
+			}
+      changeAuthorizationStatus: ^void (int status)
```

Comment:
This is indented with spaces instead of tabs, please make it consistent.

Since you're new, I recommend that you install an [EditorConfig](http://editorconfig.org/)-plugin for your text-editor, to make indentation mixups less error-prone.

Reply:
Thanks, double checked the spaces and installed Editor-Config plugin in Atom, but seems that it does not work properly. Will recheck it

Reply:
Yeah, sadly the Atom plugin for EditorConfig has a history of not working very reliably. I'm using it and it seems to work for me, but I've seen others reporting problems.